src/supports-popups.ts

```diff
@@ -31,7 +32,9 @@ function isAndroidWebview(ua?: string): boolean {
 
   ua = ua || window.navigator.userAgent;
   if (isAndroid(ua)) {
-    return androidWebviewRegExp.test(ua) && !isOperaMini(ua);
+    return (
+      androidWebviewRegExp.test(ua) && !isOperaMini(ua) && !isDuckDuckGo(ua)
```

Comment:
what are you basing the inclusion of `isDuckDuckGo` here? I was able to open popups in the Android Duck Duck Go browser just fine.

Reply:
Confirmed, indeed, I am excluding DuckDuckGo from AndroidWebViews since `supportsPopups` returns `!(isAndroidWebview(ua) || ...)`, so if I didn't make any mistakes, `supportsPopups` should return `true` for DuckDuckGo.

Reply:
Ahh, yes, of course. I should have expanded the code snippet to see where I was in the file.